extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryHandlerValueFactory.java

```diff
@@ -48,10 +53,12 @@ public static SentryOptions toSentryOptions(SentryConfig sentryConfig) {
         sentryConfig.inAppPackages
                 .map(p -> join(",", p))
                 .filter(s -> !Objects.equals(s, "*"))
-                .ifPresent(options::addInAppInclude);
+                .map(p -> p.split(","))
+                .ifPresent(inAppPackages -> Arrays.stream(inAppPackages).forEach(options::addInAppInclude));
```

Comment:
It seems we are going back and forth here (join then split). I guess, instead we should do something like this (I haven't tested this code):
```java
sentryConfig.inAppPackages
	.filter(p -> p != null)
	.filter(p -> p.size() != 1 || !Objects.equals(p.get(0), "*"))
	.ifPresent(p -> p.forEach(options::addInAppInclude));
```

Reply:
Hmmm, given it's runtime code, I would even recommend to get rid of the lambdas entirely.

We try to avoid them in runtime code as they consume more memory.

Reply:
yeah, that would be even better without lambda (that's on me as I added them in the first place :) ), @maciejwalkowiak you may also replace the others `ifPresent` under that one.

Don't forget to squash to one commit..